editor/static/css/preview.css

```diff
@@ -12,18 +12,59 @@ header {
     /*! border-bottom: 0.2rem solid hsl(0, 70%, 10%); */
     display: grid;
     color: white;
-    box-shadow: 0 0px 1px 2px hsl(0, 70%, 10%);
     z-index: 100;
-    line-height: 1.5em;
+    line-height: 0.5em;
     grid-template-columns: [row-start] 1fr repeat(4,max-content) [row-end];
-    grid-gap: 0.2em 2em;
+    grid-gap: 0.2em 0.75em;
     align-items: center;
 }
 header .description {
     flex-grow: 1;
     font-size: 1em;
     font-weight: normal;
 }
+
+
+.no-underline-link {
+    text-decoration:none;
+}
+
+.underline-text{
+    text-decoration: underline;
+}
+
+.pointer-effect{
```

Comment:
This is a hack around the fact that you want tags with this class to work like buttons. The correct thing to do is to make them buttons, and style them to look like text.